src/AddItems.js

```diff
@@ -68,7 +64,7 @@ function AddItems(props) {
             value={30}
             name="frequency"
             checked={itemFreq === 30}
-            required={formValidationVisual}
```

Comment:
We went ahead and went back to simply making the form inputs `required` based on feedback - we'll highlight the problem we were trying to solve (by tying this property to 'formValidationVisual' state) in our demo and see if we feel as a group that it's a problem that needs to be solved